red-hat-camel-k/instance/integration-platform/README.md

```diff
@@ -0,0 +1,39 @@
+# OpenShift Serverless - Knative Eventing
+
+Installs the `IntegrationPlatform` of the Red Hat Integration - Camel-k Operator.
```

Comment:
I think the IntegrationPlatform comes with the operator installation automatically. Do we really need this custom IntegrationPlatform resource?

Reply:
Ok, I'll remove the instance directory